### DISCUSSION TITLE
One Item in a column

### DISCUSSION THREAD

**Discussion by rainmanjam** [Lissy93/dashy#1578] [Q&A]
I have a collection on the left side first column I want to be by itself and have all of the others in the next 3 rows. Is there a way to edit the config so personal is solo in that row? Nothing should be under personal.
![Screenshot 2024-05-09 at 9 42 13 PM](https://github.com/Lissy93/dashy/assets/181322/64c9d785-4373-4e37-a4ca-c12dbba167a7)

**Comment by CrazyWolf13** [comment]
![image](https://github.com/Lissy93/dashy/assets/96661824/93408288-a94e-4faf-81f7-2cfad72c8879)
![image](https://github.com/Lissy93/dashy/assets/96661824/f550e06d-bd74-4e33-8eff-3e78c73c4d02)

Here you can choose the number of colums and the number of rows of each section, so you could set the Personal Section to be 1 Column and like 6 rows, depending on the screen size.
